ui/src/doc-inline/it/data_agent_1_table_agent.md

```diff
@@ -0,0 +1,34 @@
+# Per Agente
+
+Dati suddivisi *per Agente* selezionato nelle code selezionate nel 
+periodo impostato, i dati vengono raggruppati secondo la scelta fatta 
+nei filtri.
+
+I dati *per Agente* mostrano l'attività lavorativa e il carico delle 
+telefonate con cui hanno interagito per ogni agente selezionato.
+
+Le voci mostrate per i dati *per Agente* sono:
+
+- Periodo di Raggruppamento
+- Agente
+- Numero Coda
+- Nome Coda
+- Giorni Lavorativi (giorni con attività nel periodo selezionato)
+- Login (tempo totale di login in coda) 
+- Pausa (tempo totale di pausa in coda)
+- Dopo Chiusura (il totale del tempo che ogni coda può concedere 
```

Comment:
This section should use a better formatting.
Something like:
```
- `Periodo di Raggruppamento`
...
- `Dopo Chiusura`: il totale del tempo che ogni coda può concedere 
```

Or, as alternative:
```
- `Periodo di Raggruppamento`
...
- *Dopo Chiusura*: il totale del tempo che ogni coda può concedere 
```